### ISSUE TITLE
official enfuse error

### ISSUE THREAD

**Issue by dlaciv12** [darktable-org/lua-scripts#208]
Running git 3.1.0+261 on Ubuntu.
From `darktable -d lua` I get:
`LUA ERROR : /home/dave/.config/darktable/lua/official/enfuse.lua:71: attempt to concatenate a nil value (local 'version')`
Let me know if you need any other info.

**Comment by wpferguson** [comment]
Thanks, I'll check it out....

**Comment by wpferguson** [comment]
What version of enfuse are you running, and are you on MacOS or Linux?

**Comment by dlaciv12** [comment]
Version 4.2 on Ubuntu 19.10

**Comment by supertobi** [comment]
What does "enfuse --version" return in the command line?

**Comment by supertobi** [comment]
And as a fast workaround you can replace line 70:
version = string.match(f, "enfuse (%d.%d)")
with:
version = "4.2"

**Comment by wpferguson** [comment]
I put a check in to make sure there was an enfuse executable before doing anything else.  Then I ran it to test, after I renamed the executable so that it couldn't be found, and got the same crash on the version check.  I did a check on my darktablerc file and found the line 

`lua/executable_paths/enfuse=/home/bill/.config/darktable`

check_if_bin_exists looks for this entry before using other methods to find the executable.  I do a check to see if the file exists and if it's executable, so I probably should make sure it's not a directory :).  

The fix if that happens is easy, just start tools/executable_manager and clear the executable path for enfuse and that should fix it.  @dlaciv12,  let me know if that was the problem.
